### PR TITLE
adjust dependencies

### PR DESCRIPTION
**Pull request by ALegchilov**

### CODE REVIEW

File: CHANGELOG.MD [hunk 1]
```diff
@@ -9,6 +9,9 @@ Check [Keep a Changelog](http://keepachangelog.com/) for recommendations on how
 :x: - deprecation
 :pencil: - chore
 
+## 2.0.1
+- :pencil: adjust dependencies
```
Comment (AlexGalichenko) [inline CHANGELOG.MD:13]: rocket :)

File: package.json [hunk 2]
```diff
@@ -26,20 +26,20 @@
     "node": ">=18"
   },
   "dependencies": {
+    "@cucumber/cucumber": "^11.0.1",
+    "@qavajs/memory": "^1.9.0",
+    "@qavajs/validation": "^0.10.0",
+    "@types/chai": "^4.3.19",
     "@swc/core": "^1.7.36",
+    "chai": "^4.5.0",
```
Comment (AlexGalichenko) [inline package.json:34]: remove chai

File: package.json [hunk 3]
```diff
@@ -26,20 +26,20 @@
     "node": ">=18"
   },
   "dependencies": {
+    "@cucumber/cucumber": "^11.0.1",
+    "@qavajs/memory": "^1.9.0",
+    "@qavajs/validation": "^0.10.0",
+    "@types/chai": "^4.3.19",
```
Comment (AlexGalichenko) [inline package.json:32]: remove types/chai

**Review by AlexGalichenko — CHANGES_REQUESTED** [review]

File: CHANGELOG.MD (hunk 1 above)
Comment (ALegchilov) [reply CHANGELOG.MD]: To me it is a bugfix rather, but let it be a 🚀

**Review by AlexGalichenko — APPROVED** [review]

**Pull request merged**